### ISSUE TITLE
Support for Kobo Clara 2E?

### ISSUE THREAD

**Issue by CodeShakingSheep** [Bill13579/buck#1]
Hi @Bill13579 ,
awesome project! I'm looking for an audio player for my Kobo Clara 2E for side-loaded audio books. Do you know of anybody who has tested buck with Kobo Clara 2E?

Regarding installation: I have already installed KoReader on my device. So, Nickel Menu is installed too. So, I'd only have to do step 2, 4 and 5, right?

**Comment by Bill13579** [comment]
I don't have a Clara 2E so I can't personally test support for it. Yes I think you can skip 1 and 3. It might be a bit finicky to get everything set up, but best of luck!

**Comment by CodeShakingSheep** [comment]
Alright, I just followed the setup steps 2, 4 and 5. It went smoothly. There were no error messages displayed. However, how do I access it now that it's installed? Should there be a Nickel Menu entry for buck? I still only have the "KOReader" entry there. When I click on my side-loaded audios (.mp3z archives) they are still being opened with the rudimentary Kobo dev audio player. How can I get them to open with buck? Thanks for support.

Edit: I also just copied a plain mp3 file to Kobo's root directory for testing. However, I cannot see it on my Kobo.

Edit 2: Not sure if you got notified about this comment as the issue is already closed. Could you kindly give me some hints @Bill13579 ?

**Comment by YoraiLevi** [comment]
@CodeShakingSheep It has been a while since this issue was posted.
I tried using buck on a Kobo Libra Color and couldn't get it to work.
I fiddled a little over the telnet console and it seems that at least for libra color bluetooth driver deamon doesn't initialzie whatsoever so no bluealsa device exists.

I don't like deadends :(

**Comment by CodeShakingSheep** [comment]
I experimented with it again as well. Found that the music files need to be in `/mnt/onboard/music` as plain `mp3` files (not `mp3z` archives). Then they are being picked up by the player at least. When I attempt to open the player from NickelMenu I see a popup `Successfully started process with PID <PID>` and the following messages:

```
* Cataloging...
* Rendering the Table of Contents...
(this might take a while)
* Done! 
```

Nothing more is happening. So, the player actually never opens on my device. Does the player actually open on your Kobo @YoraiLevi ?
